pytest/functional/test_view_functions.py

```diff
@@ -0,0 +1,258 @@
+import json
```

Comment:
beneath pytest/functional/ I've created subfolders that correspond to Django apps. This may have occured after you created this, as the branch has been open a while.

Reply:
Sounds good. I will move the model aggregation tests to a new folder (hs_file_types).